### ISSUE TITLE
Handle of bracket. Plugin "crashed" after inserting brackets

### ISSUE THREAD

**Issue by pavo141** [simonsymhoven/data-reporting-plugin#34]
Plugin stops to Show Charts, after a bracket is given as "id". 
JSON:
```
{
      "id": "Units_(zusammengesetzte_Werte)",
      "result": {
        "incorrect": 67,
        "manually": 0,
        "accurate": 32
      }
```
      
With brackets:
![Screenshot 2022-06-17 at 09 46 13](https://user-images.githubusercontent.com/36238689/174252791-49cd38cf-49ec-4738-9b1f-28d4b0924947.png)

Without brackets:

![Screenshot 2022-06-17 at 09 49 46](https://user-images.githubusercontent.com/36238689/174252871-8640d386-d611-49d8-b2d2-d459dafc1c97.png)

**Comment by simonsymhoven** [comment]
@pavo141 
This is due to the meta charcters in JavaScript. The ids are used to assign an id to the respective charts. This id is searched by the echarts-api plugin via jQuery. 

Probably it makes most sense to fix this in the echarts-api plugin by escaping the passed chartDivId..

**Comment by simonsymhoven** [comment]
will be fixed with https://github.com/jenkinsci/echarts-api-plugin/pull/239.

**Comment by simonsymhoven** [comment]
closed by #40
